### PR TITLE
tests: add test for getting region from zone

### PR DESCRIPTION
**Pull request by ajayk**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

 /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -143,14 +143,14 @@ func ConvertLabelsStringToMap(labels string) (map[string]string, error) {
...
 func GetRegionFromZone(location string) (string, error) {
 	tokens := strings.Split(location, "-")
 	if len(tokens) != 3 {
-		return "", fmt.Errorf("Failed to parse location %v", location)
+		return "", fmt.Errorf("failed to parse location %v", location)
 	}
-
-	tokens = tokens[:2]
-	return strings.Join(tokens, "-"), nil
+	return strings.Join(tokens[0:len(tokens)-1], "-"), nil
```
Comment (leiyiz) [inline pkg/util/util.go:153]: is it needed here to do `len(tokens)-1` when we already know that the length has to be 3?

Comment (ajayk) [reply pkg/util/util.go:153]: i don't i think its needed ,probably this would make it better `strings.Join(token[0:2],"-")` ?

**Review by leiyiz — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**